app/models/concerns/taxonomix.rb

```diff
@@ -19,6 +19,7 @@ module Taxonomix
     scoped_search :relation => :organizations, :on => :id, :rename => :organization_id, :complete_enabled => false, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
 
     dirty_has_many_associations :organizations, :locations
+    audit_associations :organizations, :locations
```

Comment:
I think this will cause a bunch of warnings for resources that include taxonomix and are not audited, such as `Audit`. Perhaps add a condition here?

Reply:
I was afraid of same thing, but it's only being logged once when the rails environment starts (when classes are defined) and honestly I think we shouldn't have such resources. If something is worth of assigning to organizations, such changes should be visible in audit log.

On my setup with multiple plugins I only got these

```
2018-03-22T14:35:00 [W|sql|] ignoring associations organizations, locations audit definition for Audited::Audit, the resource is not audited
2018-03-22T14:35:02 [W|sql|] ignoring associations organizations, locations audit definition for Hostgroup, the resource is not audited
2018-03-22T14:35:09 [W|sql|] ignoring associations organizations, locations audit definition for Container, the resource is not audited
2018-03-22T14:35:09 [W|sql|] ignoring associations organizations, locations audit definition for DockerContainerWizardStates::Preliminary, the resource is not audited
```

which actually revealed a fact we don't audit host group correctly (the definition comes from NestedAncestryCommon concern), few resources in docker plugin and Audit record itself :-)

Reply:
Ok, i guess we can live with a permanent  warning the audits aren't audited